pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -491,6 +487,15 @@ func MakeSegmentStoreExternalServices(p *api.PravegaCluster) []*corev1.Service {
 				},
 			},
 		}
+		if strings.EqualFold(p.Spec.Pravega.SegmentStoreExternalTrafficPolicy, "cluster") == true {
```

Comment:
change "cluster" to "Cluster" to be consistent with k8s documentation

Reply:
Done